fog/view/server/src/config.rs

```diff
@@ -67,6 +67,30 @@ pub struct MobileAcctViewConfig {
     /// Postgres config
     #[clap(flatten)]
     pub postgres_config: SqlRecoveryDbConnectionConfig,
+
+    /// Determines which group of TxOuts the Fog View Store instance will
+    /// process.
+    #[clap(long)]
```

Comment:
Did we want to support environment variables for this argument?

Reply:
For right now, I don't because I want to wait until we meet with Ops to discuss their needs / use cases. I could see them wanting a different way to store sharding strategy etc.